src/appleseed.studio/mainwindow/mainwindow.cpp

```diff
@@ -1132,7 +1132,7 @@ void MainWindow::slot_open_cornellbox_builtin_project()
     if (!can_close_project())
         return;
 
-    const bool successful = m_project_manager.load_builtin_project("cornell_box");
+    APPLESEED_UNUSED const bool successful = m_project_manager.load_builtin_project("cornell_box");
```

Comment:
in gcc attribute unused means that the variable could be unused. It only supress the warning.